### PR TITLE
TravisCI to repo

### LINKED ISSUE #21: Add simple CI to project
- [x] Add WIP
- [ ] Shellcheck for cgi scripts 

Related to https://github.com/trikset/trik-webpanel/issues/5

### LINKED ISSUE #25: Add icon from CI about check status

### PR DESCRIPTION
**Pull request by khodand**
Closes #21 
Closes #25

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+language: minimal
+os: linux
```
Comment (khodand) [inline .travis.yml]:
```suggestion
os: linux
addons:
  apt:
    packages:
      - realpath
```

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  - export PATH=$(realpath "shellcheck-${scversion}"):$PATH
+  - shellcheck --version
+  - shellcheck --help || true
+#TODO: 
+# 1. add `-s sh`
+# 2. remove `-e 2154` and other disabled warnings
+# 3. add `-o all`
+# 4. change to `-S style`
+
+  - 'find www/cgi-bin/* | xargs file | grep shell | cut -f 1 -d : | xargs shellcheck -S warning -W 20 -f gcc --color=always -e 2154 -e 2034 -e 1090 -e 2153'
```
Comment (iakov) [inline .travis.yml]:
```suggestion
  - 'find . -executable -type f  | xargs shellcheck -S style -W 20 -f gcc --color=always'
```

File: .travis.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  - export PATH=$(realpath "shellcheck-${scversion}"):$PATH
+  - shellcheck --version
+  - shellcheck --help || true
+#TODO: 
+# 1. add `-s sh`
+# 2. remove `-e 2154` and other disabled warnings
+# 3. add `-o all`
+# 4. change to `-S style`
+
+  - 'find . -executable -type f  | xargs shellcheck -S style -W 20 -f gcc --color=always'
```
Comment (iakov) [inline .travis.yml]:
```suggestion
  - 'find . -executable -type f  -not -path "./.git*" | xargs shellcheck -S style -W 20 -f gcc --color=always'
```

File: .travis.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  - export PATH=$(realpath "shellcheck-${scversion}"):$PATH
+  - shellcheck --version
+  - shellcheck --help || true
+#TODO: 
+# 1. add `-s sh`
+# 2. remove `-e 2154` and other disabled warnings
+# 3. add `-o all`
+# 4. change to `-S style`
+
+  - 'find . -executable -type f  -not -path "./.git*" | xargs shellcheck -S style -W 20 -f gcc --color=always'
```
Comment (khodand) [inline .travis.yml]:
```suggestion
  - 'find www/cgi-bin/ -executable -type f  -not -path "./.git*" | xargs shellcheck -S style -W 20 -f gcc --color=always'
```

Comment (khodand) [inline .travis.yml]:
```suggestion
  - 'find . -executable -type f  -not -path "./.git*" -not -path "./shellcheck" | xargs shellcheck -S style -W 20 -f gcc --color=always'
```

**Comment by anastasiia-kornilova** [comment]
Good job

**Pull request merged**